src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { Link, useParams } from 'react-router-dom';
+import cn from 'classnames';
+import { Person } from '../../types';
+
+type Props = {
+  person: Person;
+};
+
+export const PersonLink: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const isWoman = person.sex === 'f';
+
+  return (
+    <tr
+      data-cy="person"
+      className={cn({
+        'has-background-warning': slug === person.slug,
```

Comment:
Ensure that the `slug` property is always a string. If there's any chance it could be `undefined` or another type, this comparison could lead to unexpected behavior.